tempesta_fw/str.c

```diff
@@ -59,36 +59,42 @@ tfw_str_del_chunk(TfwStr *str, int id)
 static TfwStr *
 __str_grow_tree(TfwPool *pool, TfwStr *str, unsigned int flag, int n)
 {
+	TfwStr *a = NULL;
+
+	TFW_DBG("str_grow_tree:start:f:%d;f:%d;d:%s;\n", str->flags, flag,
+		str->data);
 	if (str->flags & flag) {
```

Comment:
Yeah, it seems the only place where we need `TfwStr->__flags` and `__TFW_STR_COMPOUND`